docs/DeveloperGuide.md

```diff
@@ -521,330 +523,322 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is  `SudoHR` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Add a new employee**
+**Use case: UC1 - Adding a employee**
 
 **MSS**
 
-1.  User requests to list employees
-2.  SudoHR shows a list of employees
-3.  User requests to delete a specific employee in the list
-4.  SudoHR deletes the employee
+1.  User requests to add a new employee to SudoHR.
+2.  SudoHR adds the employee.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. There is already an employee in SudoHR with the same eid, phone number or email as the new employee added.
 
-  Use case ends.
+    * 1a1. SudoHR shows an error message.
 
-* 3a. The given index is invalid.
+      Use case resumes at step 1.
 
-    * 3a1. SudoHR shows an error message.
+* 1b. The given command has invalid inputs.
 
-      Use case resumes at step 2.
+    * 1b1. SudoHR shows an error message.
 
-**Use case: UC1 - Create a department**
+     Use case resumes at step 1.
 
-**MSS:**
-1. User requests to create a department.
-2. SudoHR creates the department.
+* 1c. The given command has missing inputs.
 
-    Use case ends.
+    * 1c1. SudoHR shows an error message.
 
-**Extensions**
+      Use case resumes at step 1.
 
-* 1a. The given argument is invalid.
 
-    * 1a1. SudoHR shows an error message.
 
-      Use case resumes at step 1.
+**Use case: UC2 - Delete a employee**
 
-**Use case: UC2 - Add employee to department**
-**MSS:**
-1. User lists department
-2. User requests to add employee to the department.
-3. SudoHR adds employee to the department.
-4. Use case ends.
+**MSS**
 
-**Extensions**
+1.  User requests SudoHR to __list all employees(UC5)__.
+2.  User requests to delete a specific employee with a specific eid.
+3.  SudoHR deletes the employee.
```

Comment:
I think listing shouldnt be in the use cases. We technically dont need to list the employees to remove an employee. Previously it was relevant for AB3 since AB3 go by indexes which are dynamic.

Reply:
ok noted

Reply:
Resolved